src/main/createWindow.js

```diff
@@ -36,6 +36,8 @@ const createWindow = (pathname, options = {}) => {
     win.show()
 
     if (pathname) {
+      // TODO: Check whether the file has a valid extension and isn't a directory.
+
```

Comment:
This is a very good feature, Before you MERGE this PR, can you finish this **TODO**.

Reply:
:+1: